bchrpc/pb/bchrpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.25.0
-// 	protoc        v3.12.4
+// 	protoc        v3.3.0
```

Comment:
Need to use the same version as bchd master 3.12.4

Reply:
I just updated it to `v3.14.0` which is newer than in BCHD master branch. If they want to keep the exact version on merge they can just run `regen.sh`